components/EditableDateField/index.tsx

```diff
@@ -1,18 +1,30 @@
-import InputMask, { Props } from 'react-input-mask'
+import { PatternFormat } from 'react-number-format'
```

Comment:
react-input-mask was not working with react strict mode. this lib is no longer supported and can be replaced by react-number-format. we just made this change to KanmonOS.